### PR TITLE
Multiple appointments

### PR DESCRIPTION
**Pull request by dwhenry**
-- DO NOT MERGE --

- [x] related DB migration PR #210 has been run
- [ ] golive date - 20th July

This code uses alias_attribute to have two radio lists for a single field. The second radio list takes precedence over the first if a value in it is selected.

### CODE REVIEW

File: db/migrate/20160712095513_add_number_of_appointments_to_appointment_summary.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddNumberOfAppointmentsToAppointmentSummary < ActiveRecord::Migration
+  def change
+    add_column :appointment_summaries, :number_of_previous_appointments, :string, null: false, default: ''
```
Comment (benlovell) [inline db/migrate/20160712095513_add_number_of_appointments_to_appointment_summary.rb]: As I said over on #210 this should be numeric.

Comment (dwhenry) [inline db/migrate/20160712095513_add_number_of_appointments_to_appointment_summary.rb]: Have now converted the field to a boolean with a "hidden" zero field

File: app/views/appointment_summaries/new.html.erb [hunk 2]
```diff
@@ -106,6 +106,67 @@
...
 
+    <div class="form-group if-no-js-hide">
+      <fieldset>
+        <legend>
+          <span>Is this the callers first appointment?</span>
+        </legend>
+        <div class="radio">
+          <label>
+            <%= radio_button_tag :first_appointment, 'yes',
+                                 f.object.number_of_previous_appointments == 0,
```
Comment (benlovell) [inline app/views/appointment_summaries/new.html.erb]: `.zero?` instead?

File: app/views/appointment_summaries/new.html.erb [hunk 3]
```diff
@@ -106,6 +106,67 @@
...
+                                 class: 't-first-appointment-yes js-first-appointment' %>
+            Yes
+          </label>
+        </div>
+        <div class="radio">
+          <label>
+            <%= radio_button_tag :first_appointment, 'no',
+                               f.object.number_of_previous_appointments > 0,
```
Comment (benlovell) [inline app/views/appointment_summaries/new.html.erb]: Sorry, or even `.positive?` to be more specific.

Comment (dwhenry) [inline app/views/appointment_summaries/new.html.erb]: 👍  done.

**Comment by benlovell** [comment]
:shipit:

**Pull request closed without merging**